README.md

```diff
@@ -63,22 +63,21 @@ where the team's handle is tagged in the alert message.
 
 #### Usage Guidelines:
 
-Apply the `alert_routing_key` annotation to alerts in order to get notified about them.
+Apply the `alert_team_handle` and `team` annotations to SLO alerts in order to get notified about them.
   
-#### How to apply the `alert_routing_key` Annotation:
+#### How to apply the `alert_team_handle` Annotation:
 
-Apply the `alert_routing_key` key to the annotations section of any alerting rule,
-with one of the team's namespaces as its value, or the team's name.
+Apply the `alert_team_handle` key to the annotations section of any alerting rule,
+with the relevant team's Slack handle.
+The format of the Slack handle is: `<!subteam^-team_id->` (e.g: `<!subteam^S041261DDEW>`).
```

Comment:
What's `team-id`? Is that the Slack group ID? We should use the same name used in Slack and maybe also explain how to retrieve it.